temba/settings_common.py

```diff
@@ -697,6 +709,7 @@ def traces_sampler(sampling_context):  # pragma: no cover
         "flows.ruleset.*",
         "flows.flowrun_delete",
         "flows.flowimage.*",
+        "flows.flowtemplate.*",
```

Comment:
Per what I understood, the flow templates feature management would be accessible only by CCL admins, not by account admins. So I think it should be available only for "Customer Support" or super users.